internal/delete/delete_item_action_handler.go

```diff
@@ -160,10 +160,10 @@ func (ctx *Context) getApplicableActions(groupResource schema.GroupResource, nam
 }
 
 // unmarshal reads a JSON file into an Unstructured data object.
-func (ctx *Context) unmarshal(filePath string) (*unstructured.Unstructured, error) {
+func unmarshal(fs filesystem.Interface, filePath string) (*unstructured.Unstructured, error) {
```

Comment:
Now that we have `archive.Unmarshal` this can be removed?

Reply:
I forgot I made this comment. So, LGTM pending this change 😄

Reply:
Thought I removed it, but had missed it. Updated!